lib/elixir/lib/version.ex

```diff
@@ -119,6 +119,27 @@ defmodule Version do
     defstruct [:source, :matchspec, :compiled]
 
     @opaque t :: %__MODULE__{source: String.t(), matchspec: :ets.match_spec(), compiled: boolean}
```

Comment:
I think the correct typespec for `:matchspec` is `:ets.match_spec() | :ets.comp_match_spec()`

Reply:
Good catch :+1: will update